chickadee/processes/wps_CI.py

```diff
@@ -0,0 +1,144 @@
+from pywps import Process, ComplexInput, ComplexOutput, LiteralInput, FORMATS
+from pywps.app.Common import Metadata
+from pywps.app.exceptions import ProcessError
+
+from wps_tools.utils import log_handler
+from wps_tools.io import log_level
+from chickadee.utils import logger, get_ClimDown
+
+
+class CI(Process):
+    def __init__(self):
+        self.status_percentage_steps = {
+            "start": 0,
+            "process": 10,
+            "build_output": 95,
+            "complete": 100,
+        }
+        inputs = [
+            log_level,
+            ComplexInput(
+                "gcm_file",
+                "GCM NetCDF file",
+                abstract="Filename of GCM simulations",
+                min_occurs=1,
+                max_occurs=1,
+                supported_formats=[FORMATS.NETCDF],
+            ),
+            ComplexInput(
+                "obs_file",
+                "Observations NetCDF file",
+                abstract="Filename of high-res gridded historical observations",
+                min_occurs=1,
+                max_occurs=1,
+                supported_formats=[FORMATS.NETCDF],
+            ),
+            LiteralInput(
+                "varname",
+                "Variable to Downscale",
+                abstract="Name of the NetCDF variable to downscale (e.g. 'tasmax')",
+                min_occurs=1,
+                max_occurs=1,
+                data_type="string",
+            ),
+            LiteralInput(
+                "out_file_create",
+                "Output NetCDF File",
+                abstract="Filename to create with the climate imprint outputs",
+                min_occurs=0,
+                max_occurs=1,
+                data_type="string",
+            ),
+            ComplexInput(
+                "out_file_overwrite",
+                "Output NetCDF File",
+                abstract="Filename to overwrite with the climate imprint outputs",
+                min_occurs=0,
+                max_occurs=1,
+                supported_formats=[FORMATS.NETCDF],
+            ),
```

Comment:
Not sure if this is handled somewhere, but since there are both optional what happens if a user gives neither? Do we have a check to ensure at least one of these have been provided?